setup/setup.sh

```diff
@@ -6,6 +6,7 @@ customized_files=(
   'deploy-exclude-list.txt'
   'gulpfile.js'
   'package.json'
+  'backup-db.sh'
```

Comment:
No. This just copies this backup shell script into the root for users to customize.